packages/experience-builder-sdk/src/core/preview/assemblyUtils.spec.ts

```diff
@@ -84,44 +80,104 @@ describe('resolvePattern', () => {
 
       const result = resolvePattern({
         node: assemblyNode,
+        rootPatternParameters: {},
+        parentPatternRootNodeIdsChain: [],
         entityStore,
-        parentParameters: {},
-        patternRootNodeIdsChain: '',
       });
 
       expect(result.children).toHaveLength(1);
       expect(result.children[0].children).toHaveLength(1);
       // This will be exactly like in the definition as the instance doesn't define a value for the ComponentValue
-      expect(result.children[0].children[0]).toEqual(
-        assemblyEntry.fields.componentTree.children[0].children[0],
-      );
+      expect(result.children[0].children[0]).toEqual({
+        ...assemblyEntry.fields.componentTree.children[0].children[0],
+        displayName: undefined,
+        parameters: undefined,
+        pattern: {
+          nodeIdOnPattern: 'test-custom-component-id',
+          parentPatternNodeId: undefined,
+          prefixedNodeId: 'test-custom-component-id',
+        },
```

Comment:
wanted to point out that I decided to keep these properties just in case, but they aren't used anywhere. So we can clean them up. I'm happy to hear your opinions